vcbuild.bat

```diff
@@ -197,6 +197,7 @@ if defined link_module      set configure_flags=%configure_flags% %link_module%
 if defined i18n_arg         set configure_flags=%configure_flags% --with-intl=%i18n_arg%
 if defined config_flags     set configure_flags=%configure_flags% %config_flags%
 if defined target_arch      set configure_flags=%configure_flags% --dest-cpu=%target_arch%
+if defined debug_nghttp2    set configure_flags=%configure_flags% --debug-nghttp2
```

Comment:
Does it is needed? Looks like line 142 does everything needed
> if /i "%1"=="debug-nghttp2" set debug_nghttp2=1&goto arg-ok

Reply:
Apparently it is. We need to pass the `--debug-nghttp2` option to `configure` so it reaches `gyp`.